app/models/appointment.rb

```diff
@@ -0,0 +1,4 @@
+class Appointment < ApplicationRecord
+  belongs_to :user
+  belongs_to :doctor
```

Comment:
Because there's no validation to check for a doctor and user, if you try to create an appointment without one, you get a Rails error.